ui/src/components/WMWorkerPoolEditor/index.jsx

```diff
@@ -397,6 +402,7 @@ export default class WMWorkerPoolEditor extends Component {
               value={JSON.stringify(workerPool.config, null, 2)}
               onChange={this.handleEditorChange}
               lint
+              options={options}
```

Comment:
Good job on making it work :) I think it would make sense to move this upstream (i.e., in https://github.com/mozilla-frontend-infra/components) since it would make the CodeEditor more accessible. Once we move it there, we could release a new version of `@mozilla-frontend-infra/components` and then bump the version in this repo to receive the new changes. Feel free to create a PR directly there :)

Reply:
How is it going @devnoorfatima?